sphinxcontrib/needs/directives/needextend.py

```diff
@@ -121,9 +121,9 @@ def process_needextend(app, doctree, fromdocname):
 
                         # else it must be a normal string
                         else:
-                            # If content is already stored, we need to add some whitespace
+                            # If content is already stored, we need to add some whitespace and comma
                             if need[option_name]:
-                                need[option_name] += " "
+                                need[option_name] += ", "
```

Comment:
But what if I just want to add a string?
"," gets added now everywhere, for each field.

We only need it for comma-separated fields, which are tag and all links.
Everything else is a String-based field.

So this needs to work:
```

.. req:: My req
   :id: TEST_01
   :author: Me

.. needextend:: TEST_01
   +author: and some friends
```

So "author" should be "me and some friends", whithout any comma.